qiskit/test/mock/fake_backend_v2.py

```diff
@@ -176,3 +180,40 @@ def qubit_properties(self, qubit):
         if isinstance(qubit, int):
             return self._qubit_properties[qubit]
         return [self._qubit_properties[i] for i in qubit]
+
+
+class FakeBackendSimple(BackendV2):
+    """A fake simple backend that wraps BasicAer to implement run()."""
+
+    def __init__(self):
+        super().__init__(
+            None,
+            name="FakeSimpleV2",
+            description="A fake simple BackendV2 example",
+            online_date=datetime.datetime.utcnow(),
+            backend_version="0.0.1",
```

Comment:
I am a bit confused. There seem to be two different `backend_version`. One is the integer-only versioned backend which is later referenced as `backend_interface_version`: `0` for LegacyBackend, `1` for BackendV1 and `2` for BackendV2. the other one is the `backend_version` has SemVer format like `0.0.1` that comes from `conf.json` from real backends? Am I right?

Reply:
Are you thinking of the `backend_version` entry in the BackendV1->V2 migration guide: https://qiskit.org/documentation/apidoc/providers.html#migrating-between-backend-api-versions it might be somewhere else too.

But yeah, @jlapeyre is correct as to the difference. The `version` attribute is set in the abstract class definition as a class variable and is used to indicate which version of that interface a particular backend object implements. While the `backend_version` attribute is arbitrary metadata that a backend can optionally supply when it's implemented to indicate any version information. The semver like string is just what IBM happen to uses for its backends (and that comes from the backend configuration json payload), but it can be any string as it's up to each backend. I agree the dual use of version can be confusing, but it is something that's not easy to change since this is part of a defined interface and it would require a deprecation cycle to change (and honestly I think it'll just be better to put it on the list for the inevitable `BackendV3` and wait for that)